terminal.c

```diff
@@ -1033,8 +1033,6 @@ void terminal_process_string(char *str) {
 				commands_printf("Invalid arguments\n");
 			}
 		}
-	} else if (strcmp(argv[0], "build_date") == 0) {
-		commands_printf("Build date and time: %s at %s\n", __DATE__, __TIME__);
```

Comment:
You should remove the help text too if you remove this one (line 1182 - 1183)

Reply:
Indeed.  I took the help text out.